Makefile

```diff
@@ -5,7 +5,7 @@ include newline.mk
 # Image URL to use all building/pushing image targets
 IMG ?= kfp-operator-controller
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:preserveUnknownFields=false"
+CRD_OPTIONS ?= "crd:ignoreUnexportedFields=false"
```

Comment:
As discussing directly, this change doesnt seem to matchup

Reply:
removed `ignoreUnexportedFields=false` as its the default behaviour.

In order to upgrade a CRD from v1beta1 to v1 you would need to use the `preserveUnknownFields=true` flag.
So reasons why we should not be using it:
- It was deprecated as of 1.22 which is EOL from years ago.
- All kfp custom resources are already using `apiVersion: apiextensions.k8s.io/v1` so theres no need for us to upgrade any CRDs here.

Reply:
Makes total sense to me 👍